Tests are failing

```
=== RUN   TestEmpty
--- PASS: TestEmpty (0.00s)
=== RUN   TestEmptyWithIDAndAttributes
--- PASS: TestEmptyWithIDAndAttributes (0.00s)
=== RUN   TestTwoConnectedNodes
--- FAIL: TestTwoConnectedNodes (0.00s)
    graph_test.go:34: got [digraph  {n1[label="A"];n2[label="B"];n1->n2;}] want [digraph  {node[label="A"] n1;node[label="B"] n2;n1->n2;}]
=== RUN   TestGraph_FindEdges
--- PASS: TestGraph_FindEdges (0.00s)
=== RUN   TestSubgraph
--- PASS: TestSubgraph (0.00s)
=== RUN   TestSubgraphClusterOption
--- PASS: TestSubgraphClusterOption (0.00s)
=== RUN   TestEdgeLabel
--- FAIL: TestEdgeLabel (0.00s)
    graph_test.go:72: got [digraph  {n1[label="n1"];n2[label="n2"];n1->n2[label="wat"];}] want [digraph  {node[label="n1"] n1;node[label="n2"] n2;n1->n2[label="wat"];}]
=== RUN   TestCluster
--- PASS: TestCluster (0.00s)
=== RUN   TestIndentWriter
--- PASS: TestIndentWriter (0.00s)
FAIL
FAIL	github.com/emicklei/dot	0.119s

Process finished with exit code 1
```

The generated charts seem to be valid though. So just forgot updating test cases in #2 ?
I'd recommend integrating Travis CI to get early feedback if tests fail. It's available in the GitHub Marketplace.

--- a/graph_test.go
+++ b/graph_test.go
@@ -29,7 +29,7 @@ func TestTwoConnectedNodes(t *testing.T) {
 	n1 := di.Node("A")
 	n2 := di.Node("B")
 	di.Edge(n1, n2)
-	if got, want := flatten(di.String()), `digraph  {node[label="A"] n1;node[label="B"] n2;n1->n2;}`; got != want {
+	if got, want := flatten(di.String()), `digraph  {n1[label="A"];n2[label="B"];n1->n2;}`; got != want {
 		t.Errorf("got [%v] want [%v]", got, want)
 	}
 }
@@ -56,7 +56,7 @@ func TestEdgeLabel(t *testing.T) {
 	n1 := di.Node("n1")
 	n2 := di.Node("n2")
 	n1.Edge(n2, "wat")
-	if got, want := flatten(di.String()), `digraph  {node[label="n1"] n1;node[label="n2"] n2;n1->n2[label="wat"];}`; got != want {
+	if got, want := flatten(di.String()), `digraph  {n1[label="n1"];n2[label="n2"];n1->n2[label="wat"];}`; got != want {
 		t.Errorf("got [%v] want [%v]", got, want)
 	}
 }

